assets/src/infrastructure/ui/inputs/inlineEdit/InlineEditInput.tsx

```diff
@@ -33,9 +34,13 @@ const InlineEditInput: React.FC<InlineEditInputProps> = ({ inputType, onCancel,
 				}
 			},
 		};
+
+		// @ts-ignore
+		return <ChakraEditableInput {...textareaProps} variant='unstyled' />;
 	}
 
-	return <ChakraEditableInput {...textareaProps} />;
+	// @ts-ignore
+	return <ChakraEditableInput className={className} type={inputType} variant='unstyled' />;
```

Comment:
2 issues here
1. `type` is not allowed here, but at the same time in docs: https://chakra-ui.com/editable#editableinput-props
```
EditableInput composes PseudoBox so you can pass all PseudoBox props.

```
and pseudo box in docs accepts `type` prop:

![image](https://user-images.githubusercontent.com/1477453/80536319-dd643900-89aa-11ea-896f-3610017511af.png)